### PR TITLE
Quick and dirty approach to decimal numbers

### PR DESCRIPTION
**Pull request by qmac**
Adds support for basic decimal numbers in the format "xx point x x x".

Approach: during tokenization, once the token "point" is parsed, add all future tokens do a different `decimal_tokens` list. This separate list is then evaluated manually outside of the main FST. Then, add the output of the integer FST with the decimal value to get a final decimal number.

### CODE REVIEW

File: tests/test_EN_US.py [hunk 1]
```diff
@@ -24,7 +24,16 @@ def test_en_us(self):
...
-            "nine trillion eight hundred seven")
+            "nine trillion eight hundred seven",
+            "ninety nine point nine",
+            "ninety nine point nine nine nine nine nine nine nine six",
+            "fifteen point four",
+            "two thousand point five",
+            "eleven million point three three",
+            "one billion point six four",
+            "six hundred point nine nine",
+            "one point zero zero four",
```
Comment (JDongian) [inline tests/test_EN_US.py:35]: Do you want to support "oh" as a synonym for "zero"?

File: words2num/lang_EN_US.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from __future__ import division, unicode_literals, print_function
 import re
 from .core import NumberParseException, placevalue
+from decimal import Decimal, localcontext
```
Comment (JDongian) [inline words2num/lang_EN_US.py:4]: you forgot to commit this file

File: words2num/lang_EN_US.py [hunk 3]
```diff
@@ -55,7 +56,7 @@
     'octodecillion': (10**57, 'X'),
     'novemdecillion': (10**60, 'X'),
     'vigintillion': (10**63, 'X'),
-    'centillion': (10**303, 'X'),
+    'centillion': (10**303, 'X')
```
Comment (JDongian) [inline words2num/lang_EN_US.py:60]: curious what you think of this convention

File: words2num/lang_EN_US.py (hunk 2 above)
Comment (qmac) [reply words2num/lang_EN_US.py:4]: `decimal` is a built-in Python library: https://docs.python.org/3/library/decimal.html

File: words2num/lang_EN_US.py (hunk 3 above)
Comment (qmac) [reply words2num/lang_EN_US.py:60]: Purely personal preference to not have dangling commas

File: tests/test_EN_US.py (hunk 1 above)
Comment (qmac) [reply tests/test_EN_US.py:35]: Yeah sure

**Comment by qmac** [comment]
FWIW, did a brief performance comparison with [word2number](https://github.com/akshaynagpal/w2n):

screen shot 2018-08-02 at 5 34 37 pm

File: words2num/lang_EN_US.py [hunk 4]
```diff
@@ -169,8 +186,27 @@ def compute(tokens):
...
+    return float(total) if tokens else 0
+
+
 def evaluate(text):
-    tokens = tokenize(text)
+    tokens, decimal_tokens = tokenize(text)
     if not tokens:
         raise ValueError("No valid tokens in {0}".format(text))
-    return compute(tokens)
+    decimal_value = compute_decimal(decimal_tokens)
```
Comment (JDongian) [inline words2num/lang_EN_US.py]: why not just one line?

**Comment by JDongian** [comment]
I think this looks good. Should update the version number.
Maybe should move this to our private repos (?).

**Pull request merged**